### PR TITLE
use janky probly will be fixed free google translate tts api

### PR DESCRIPTION
**Pull request by harrisonchiu**
also idk if the .play() function can play files like this idk what file type the api returns.

also i am slowly accepting that i am not the 10x dev i once thought i was.
😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭
😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -197,10 +198,22 @@ client.on('message', async (message) => {
   else if (isSayCommand(message.content)) {
     const languageCode = 'en';
     const text = isSayCommand(message.content);
-    const api = `https://translate.google.com.vn/translate_tts?ie=UTF-8&q=${text}+&tl=${languageCode}&client=tw-ob`;
+
+    const curl = `curl 'https://translate.google.com/translate_tts?ie=UTF-8&q=${text}&tl=${languageCode}&client=tw-ob'`;
+    curl += " -H 'Referer: http://translate.google.com/' -H 'User-Agent: stagefright/1.2 (Linux;Android 5.0)' > tts.mp3";
+    // const api = `https://translate.google.com.vn/translate_tts?ie=UTF-8&q=${text}+&tl=${languageCode}&client=tw-ob`;
+
+    exec(curl, (error, stdout, stderr) => {
```
Comment (tonylizj) [inline index.js]: ?????

**Review by tonylizj — CHANGES_REQUESTED** [review]
fix

File: index.js (hunk 1 above)
Comment (harrisonchiu) [reply index.js]: DONT WORRY
IT IS NOW FIXED
AXIOS GET() IS USED

**Comment by harrisonchiu** [comment]
RETURN OF THE 10X DEV

**Comment by harrisonchiu** [comment]
on 2nd thought
idk the file system whatever environemtn u host in is like. it saves the mp3 in `~/tts.mp3` and then fetches it again from there. idk if it will save the right place

File: index.js [hunk 2]
```diff
@@ -196,27 +200,31 @@ client.on('message', async (message) => {
...
+        'User-Agent': 'stagefright/1.2 (Linux;Android 5.0)'
+      },
+      responseType: 'stream'
+    }).then(response => {
+      response.data.pipe(writer);
+      return finished(writer);
     });
 
     const connection = await message.member.voice.channel.join();
     connection.play('~/tts.mp3');
```
Comment (tonylizj) [inline index.js]: does this not need to be in the .then

**Review by tonylizj — CHANGES_REQUESTED** [review]

File: index.js (hunk 2 above)
Comment (harrisonchiu) [reply index.js]: what

Comment (harrisonchiu) [reply index.js]: doesnt the const connection lines run after the axios() block

**Comment by tonylizj** [comment]
ok i have no idea if this works

**Pull request merged**